src/pages/index.js

```diff
@@ -0,0 +1,331 @@
+const previewModalCloseButton = previewModal.querySelector(".modal__close-btn");
+const deleteCancelButton = document.getElementById("cancel-delete-btn");
+
+function getCardElement(data) {
+  const cardElement = cardTemplate.content
+    .querySelector(".card")
+    .cloneNode(true);
+
+  const cardNamedl = cardElement.querySelector(".card__title");
+  const cardImage = cardElement.querySelector(".card__image");
+  const likeButton = cardElement.querySelector(".card__like-btn");
+  const deleteButton = cardElement.querySelector(".card__delete-btn");
+
+  cardNamedl.textContent = data.name;
+  console.log("Card data:", data);
+  console.log("Card image element src:", cardImage.src);
+
+  console.log("Card name:", data.name);
+  console.log("Card link:", data.link);
+
+  cardImage.src = data.link;
+  cardImage.alt = data.name;
+
+  const isLikedByCurrentUser = data.isLiked === true;
+
+  if (isLikedByCurrentUser) {
+    likeButton.classList.add("card__like-btn_liked");
+  }
+
+  if (
+    Array.isArray(data.likes) &&
+    data.likes.some((user) => user._id === currentUserId)
+  ) {
+    likeButton.classList.add("card__like-btn_liked");
+  }
+
+  likeButton.addEventListener("click", (evt) => {
+    handleLike(evt, data._id);
+  });
+
+  deleteButton.addEventListener("click", () =>
+    handleDeleteCard(cardElement, data._id)
+  );
+
+  cardImage.addEventListener("click", () => {
+    openModal(previewModal);
+    previewModalCaptionEl.textContent = data.name;
+    previewModalImageEl.src = data.link;
+    previewModalImageEl.alt = data.name;
+  });
+
+  return cardElement;
+}
+
+function handleeditFormSubmit(evt) {
+  evt.preventDefault();
+
+  const submitBtn = evt.submitter;
+  setButtonText(submitBtn, true);
+
+  api
+    .editUserInfo({
+      name: editModalNameInput.value,
+      about: editModalDescriptionInput.value,
+    })
+    .then((data) => {
+      profileName.textContent = data.name;
+      profileDescription.textContent = data.about;
+      closeModal(editModal);
+    })
+    .catch(console.error)
+    .finally(() => {
+      setButtonText(submitBtn, true);
+    });
+}
+
+function handlecardFormSubmit(evt) {
+  evt.preventDefault();
+
+  const submitBtn = evt.submitter;
+  setButtonText(submitBtn, true);
+
+  api
+    .addCard({
+      name: cardModalNameInput.value,
+      link: cardModalLinkInput.value,
+    })
+    .then((cardData) => {
+      const cardElement = getCardElement(cardData);
+      cardsList.prepend(cardElement);
+      evt.target.reset();
+      disabledButton(cardSubmitBtn, settings);
+      closeModal(cardModal);
+    })
+    .catch(console.error)
+    .finally(() => {
+      setButtonText(submitBtn, true);
+    });
+}
+
+function handleAvatarSubmit(evt) {
+  evt.preventDefault();
+
+  const submitBtn = evt.submitter;
+  setButtonText(submitBtn, true);
+
+  api
+    .editAvatarInfo(avatarInput.value)
+    .then((data) => {
+      document.getElementById("image-avatar").src = data.avatar;
```

Comment:
You've already found the avatar in line 10 as `avatarImage`. Please, reuse it everywhere